bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/BitcoindRpcClient.scala

```diff
@@ -368,10 +370,10 @@ sealed trait BitcoindVersion
 object BitcoindVersion extends StringFactory[BitcoindVersion] with Logging {
 
   /** The newest version of `bitcoind` we support */
-  val newest: BitcoindVersion = V23
+  val newest: BitcoindVersion = V24
```

Comment:
I'm seeing some test failures in `node`. I'm _guessing_ that its related to bumping this version. 

If we wanted to merge this PR, i would suggest reverting this to `V23` and I can investigate why our `nodeTest` does not seem to be playing nicely with `V24`. 

I believe that will unblock you so you can use v24 while I investigate the bug in our `node` impl. 

Thoughts?

Reply:
bug is most likely related to this: https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-24.0.md#p2p-and-network-changes

Reply:
I tried debugging for a little but can't figure it out. Went ahead and reverted to v23, if you wanna take a stab at it that'd be great